interpreters/wamr/Makefile

```diff
@@ -32,22 +42,25 @@ VPATH += $(WAMR_UNPACK)/product-mini/platforms/nuttx
 
 MAINSRC = main.c
 
-PROGNAME  = iwamr
+PROGNAME  = iwasm
 PRIORITY  = $(CONFIG_INTERPRETERS_WAMR_PRIORITY)
 STACKSIZE = $(CONFIG_INTERPRETERS_WAMR_STACKSIZE)
 MODULE    = $(CONFIG_INTERPRETERS_WAMR)
 
 $(WAMR_TARBALL):
 	$(Q) echo "Downloading $(WAMR_TARBALL)"
-	$(Q) wget $(WAMR_URL)
+	$(Q) curl -O -L $(WAMR_URL)
 
 $(WAMR_UNPACK): $(WAMR_TARBALL)
 	$(Q) echo "Unpacking $(WAMR_TARBALL) to $(WAMR_UNPACK)"
-	$(Q) tar xzvf $(WAMR_TARBALL)
-	$(Q) mv wasm-micro-runtime-WAMR-$(WAMR_VERSION) $(WAMR_UNPACK)
+	$(Q) unzip $(WAMR_TARBALL)
+	$(Q) mv wasm-micro-runtime-$(WAMR_VERSION_STUB)$(WAMR_VERSION) $(WAMR_UNPACK)
 
 context:: $(WAMR_UNPACK)
 
+clean::
+	$(call DELDIR, $(WAMR_UNPACK))
+
```

Comment:
What's made during `context` should be removed during `clean_context` or `distclean`.  
If we remove it during `clean` we would always have to re-download it if we want a clean build.

Reply:
Yes, clean should be removed.

Reply:
**WAMR_UNPACK** is the directory of unpacked context, but source tarball still here.
If WAMR_VERSION changed in menuconfig, the correct source can be unpacked after `make clean`.

Reply:
OK, that requires `make distclean` after Kconfig option changed